CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+## [release-53] - 2019-08-22
```

Comment:
This doesn't seem to be link, presumably this happens magically when we create the git tag?

Reply:
Seems to look okay on the API release though.. huh  https://github.com/dxw/DataSubmissionServiceAPI/blob/59ac32d449eb976fac1133a50a6de0ecd6ca9af0/CHANGELOG.md

Reply:
Looks like a link needs to be created like https://github.com/dxw/DataSubmissionService/commit/3328c2636155b7bc11f50f8dfa31d9e3c9297621#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR284

Reply:
Thanks for catching it - my mistake.